custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -560,23 +569,56 @@ def run_fit(self):
         DrumSchemaValidationException
             Raised when model metadata validation fails.
         """
-        self._prepare_fit()
+        cli_adapter = DrumCLIAdapter(
```

Comment:
We use the DrumCLIAdapter to convert cli args (input filenames, target types, labels, etc) to dataframes, etc which will be passed to ModelAdapter. In the future, model adapter should act as a slim layer that takes in a custom task path + dataframes and runs fit/predict/score